source/creating_content/01_creating_content_page/managing_page_security.rst

```diff
@@ -0,0 +1,112 @@
+
+There are 2 permissions available in the security management of a content page:
+
+- *Browse page*: it allows the user/group of users to access the page and its content. This permission is set by default for all users added to the security management section of the page.
+- *Edit page*: it allows the user/group of users to access the back office to edit the content of the page.
+
+These permissions have different impacts depending on the chosen access policy.
+
+- *Browse page* permission:
+
+  - For a private page, the *Browse page* permission gives access to that page, hence the importance of adding users to the security management section, otherwise nobody (except users with the general domain-level *Browse all pages*) can access the page and its content.
+  - For a public page, the *Browse page* permission doesn't change anything to the access security of that page because it is already accessible by anybody who has access to the domain.
+
+- *Edit page* permission: this permission is relevant for both private and public pages and gives edition rights on the page to any user/group of users added to the security management section of the page and granted with this *Edit page* permission.
+
+Adding and deleting users or groups of users from a page
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Adding users or groups to a page
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Go to Security > User permissions or Security > Group permissions, depending on whether a user or a group will be added.
+2. Click the **Add user/group permissions** button at the bottom of the table.
+3. Select one or more users/groups of users from the list. A search bar is also available to search a user/group by name.
+4. (optional) If all added users/groups are going to be granted the *Edit page* permission as well as the default *Browse page* permission, it is possible to grant that permission now. Otherwise permissions can be granted afterward. To grant the *Edit page* permission, at the top of the "Add user/group permissions" window, in the "Grant these permissions" section:
+
+  a. Click the **Add permission** button.
+  b. Click the *Edit page* permission button.
+
+5. Click the **Add user/group permissions** button at the bottom of the Add user/group permissions window.
+6. Click the **Save** button in the top right corner.
+
+Deleting users or groups from a page
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Go to Security > User permissions or Security > Group permissions, depending on whether a user or a group will be deleted.
+2. In the table, the first column indicates the name of the user or the group of users. The second column indicates the corresponding permission(s). The third column displays the |trash-button| button. Click that |trash-button| button to delete a chosen user/group of users.
+3. Click the **Save** button in the top right corner.
+
+Managing page security permissions for users and groups of users
```

Comment:
shouldn't this be called `Managing page permissions for users and groups of users` instead ?
same below with other occurences of `page security permissions`